cmake/installing.cmake

```diff
@@ -70,6 +70,9 @@ if (UNIX AND NOT APPLE)
   endif()
 endif()
 
+# Get supported mime types
+get_property(F3D_SUPPORTED_MIME_TYPES GLOBAL PROPERTY F3D_SUPPORTED_MIME_TYPES)
```

Comment:
doesnt seems to be used

Reply:
It is used to configure `f3d.desktop` and `f3d.thumbnailer`